ironfish-rust/src/merkle_note.rs

```diff
@@ -89,8 +89,11 @@ impl MerkleNote {
             &note.commitment_point(),
             public_key,
         );
-        let mut note_encryption_keys = [0; NOTE_ENCRYPTION_KEY_SIZE];
-        aead::encrypt(&encryption_key, &key_bytes, &mut note_encryption_keys);
+        let note_encryption_keys: [u8; NOTE_ENCRYPTION_KEY_SIZE] =
```

Comment:
Not a fan of this unwrapping, but I can fix in subsequent PR to not bloat this one (update results for return types)

Reply:
Sounds good, would definitely be nice to remove the unwraps. We're basically doing the same thing in `note.rs`'s `encrypt` function that we'd want to do here, it looks like